tools/wptrunner/wptrunner/testloader.py

```diff
@@ -411,7 +411,7 @@ def make_queue(cls, tests, **kwargs):
             group.append(test)
             test.update_metadata(metadata)
 
-        for item in groups:
+        for item in sorted(groups, key=lambda x: len(x[0]), reverse=True):
```

Comment:
This change in the ordering for all grouped sources makes me nervous. Although theoretically the groups are independent and so this doesn't matter, this seems like the kind of thing that might need to be reverted independent of the rest of this change. It also seems like something that should be overrideable in derived classes rather than hardcoded into the base class.

Reply:
James, thanks for the review. Updated as requested, can you pls review again? thanks!